parts/kubernetesmasteraddons-heapster-deployment.yaml

```diff
@@ -40,11 +40,9 @@ spec:
         - "--extra-cpu=0.5m"
         - "--memory=140Mi"
         - "--extra-memory=4Mi"
-        - "--threshold=5"
```

Comment:
Okay. Let's split the resizer change out into a different PR and make this PR simply updating the container image